<a href="https://colab.research.google.com/github/cornel-kim/samples/blob/DataScienceColab/test.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

working with data science colab

In [2]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sbn
customer = pd.read_csv("http://modcom.co.ke/datasets/Mall_Customers.csv")
print(customer)

     CustomerID  Gender  Age  Annual Income  Spending Score
0             1    Male   19             15              39
1             2    Male   21             15              81
2             3  Female   20             16               6
3             4  Female   23             16              77
4             5  Female   31             17              40
..          ...     ...  ...            ...             ...
195         196  Female   35            120              79
196         197  Female   45            126              28
197         198    Male   32            126              74
198         199    Male   32            137              18
199         200    Male   30            137              83

[200 rows x 5 columns]


describe data
If one of the count is lower then there are empties
The average spending score is 50.2 minimum is 1 and max 99
the average age is 38.85 and the maximum age is 70, min 18
the annual income has a mean of 60.56, min of 15 and max 137




In [3]:
customer.describe()

CustomerID         Age  Annual Income  Spending Score
count  200.000000  200.000000     200.000000      200.000000
mean   100.500000   38.850000      60.560000       50.200000
std     57.879185   13.969007      26.264721       25.823522
min      1.000000   18.000000      15.000000        1.000000
25%     50.750000   28.750000      41.500000       34.750000
50%    100.500000   36.000000      61.500000       50.000000
75%    150.250000   49.000000      78.000000       73.000000
max    200.000000   70.000000     137.000000       99.000000